extensions/interactions/base.py

```diff
@@ -89,6 +89,11 @@ class BaseInteraction(object):
     # containing this interaction. These should correspond to names of files in
     # feconf.DEPENDENCIES_TEMPLATES_DIR. Overridden in subclasses.
     _dependency_ids = []
+    # Additional JS library dependencies that should be loaded in pages
```

Comment:
Wait, I don't understand. I was agreeing with your earlier comment -- we should store them in the same folder, in the same attr of the interaction class and include both of them whenever the interaction is present. What I don't know for certain is whether any changes are needed or not, but I expect they'd likely be minimal.